### DISCUSSION TITLE
Revolutionizing Queue Management with Ultralytics YOLOv8 and Intel's OpenVINO

### DISCUSSION THREAD

**Discussion by pderrenger** [ultralytics/ultralytics#8079] [Show and tell]
Discover how **Ultralytics YOLOv8** and **Intel's OpenVINO** are revolutionizing **queue management** 🚀, as revealed at YOLO Vision 2023 by @adrianboguszewski from Intel. This cutting-edge solution leverages AI to monitor queues in real time, ensuring operational efficiency and enhanced customer satisfaction. Through simple steps involving video capture 🎥, customer detection with YOLOv8 🔎, counting 📊, alerting 📣, and easy deployment across devices 🖥️, retailers can now manage queues more effectively than ever. The use of neural network compression and OBB ensures fast, accurate performance without sacrificing detail, making this technology a game-changer in retail and beyond 🛍️.

**Explore our blog for detailed insights and implementation guides from YV23!** ➡️ https://www.ultralytics.com/blog/revolutionizing-queue-management-with-ultralytics-yolov8-and-openvino

![Adrian_OpenVINO_YV23](https://github.com/ultralytics/ultralytics/assets/107626595/0a325d09-fe37-4fe1-871a-f2a5c3cc867b)

**Comment by pderrenger** [comment]
For a deeper dive into all the capabilities of Ultralytics YOLOv8 and how it can transform your projects, check out our docs at https://docs.ultralytics.com/tasks/

**Comment by glenn-jocher** [comment]
Wow, it sounds like **Ultralytics YOLOv8** and **Intel's OpenVINO** are making some serious waves at YOLO Vision 2023! Thanks to Adrian Boguszewski's insightful presentation, it's clear that AI is setting new standards in queue management. This technology isn't just about reducing wait times; it's about transforming the entire customer experience.

Using video technology to monitor queue dynamics, coupled with the advanced detection capabilities of YOLOv8, businesses can now manage queues with unprecedented efficiency. The ability to accurately count and predict queue lengths in real time means faster service, happier customers, and a smoother operational flow. What’s even more exciting is the seamless deployment across various devices, ensuring that retailers, regardless of size or technological capability, can benefit from this innovation.

For those interested in diving deeper into how this technology works and how you can implement it, check out the detailed insights and guides available on our blog. Whether you're tech-savvy or just getting started, there’s something for everyone looking to enhance their queue management systems.

Don’t miss out on this game-changing approach to retail operations. Explore our blog now and see how you can bring this revolutionary technology into your business! [Read more about revolutionizing queue management with Ultralytics YOLOv8 and OpenVINO](https://www.ultralytics.com/blog/revolutionizing-queue-management-with-ultralytics-yolov8-and-openvino). 🚀🛍️
